config/logging.yml

```diff
@@ -27,14 +27,32 @@ log_cache_gateway:
 log_cache_syslog:
   crt: #@ data.values.log_cache_syslog.crt
   key: #@ data.values.log_cache_syslog.key
+
+log_cache_external:
+  crt: #@ data.values.log_cache_external.crt
+  key: #@ data.values.log_cache_external.key
+
+client:
+  id: #@ data.values.log_cache_client.id
+  secret: #@ data.values.log_cache_client.secret
 #@ end
 
 #@ logging = library.get("github.com/cloudfoundry/cf-k8s-logging")
 --- #@ template.replace(logging.with_data_values(logging_values()).eval())
 
+---
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: system-ca
+  namespace: #@ data.values.system_namespace
+data:
+  tls.crt: #@ data.values.system_certificate.crt
+
 #@overlay/match by=overlay.subset({"kind":"Service", "metadata":{"name": "log-cache"}})
```

Comment:
since you are bumping log-cache why dont we also fix http -> https change so that this overlay doesnt have to be here?

Reply:
Sure, we can look into doing that.